pkg/utils/secrets/manager/manager.go

```diff
@@ -205,9 +209,16 @@ func (m *manager) mustAutoRenewSecret(secret corev1.Secret) (bool, error) {
 		return false, err
 	}
 
+	renewAfterValidityPercentage := 80
+	if secret.Labels[LabelKeyRenewAfterValidityPercentage] != "" {
+		if value, err := strconv.Atoi(secret.Labels[LabelKeyRenewAfterValidityPercentage]); err == nil && value > 0 && value < 100 {
```

Comment:
Shouldn't we return an error when we cannot parse the label, or when its value is not within the boundaries? This is what we do for `issued-at` and `valid-until` a few lines above.